documentation/CAMARA-Security-Interoperability.md

```diff
@@ -0,0 +1,277 @@
+# CAMARA Security and Interoperability Profile
+
+## Table of Contents
+
+   * [Introduction](#introduction)
+   * [Audience](#audience)
+   * [Conventions](#conventions)
+   * [General Considerations](#general-considerations)
+      + [Transport Security](#transport-security)
+   * [OIDC Authorization Code Flow](#oidc-authorization-code-flow)
+      + [Cross-Site Request Forgery Protection](#cross-site-request-forgery-protection)
+   * [Client-Initiated Backchannel Authentication Flow](#client-initiated-backchannel-authentication-flow)
+      + [Optional Parameters](#optional-parameters)
+      + [Authentication Request](#authentication-request)
+   * [Format of `login_hint`](#format-of-login_hint)
+   * [Offline Access](#offline-access)
+         - [Refresh Token Issuance](#refresh-token-issuance)
+      + [Refresh Token Usage](#refresh-token-usage)
+         - [Refresh Token Usage Error Response](#refresh-token-usage-error-response)
+      + [Refresh Token Security](#refresh-token-security)
+   * [Client Credentials Flow](#client-credentials-flow)
+   * [Handling of acr_values](#handling-of-acr_values)
+   * [Access Token Request](#access-token-request)
+   * [The Scope Parameter](#the-scope-parameter)
+   * [Missing "openid" scope](#missing-openid-scope)
+   * [Purpose](#purpose)
+   * [ID Token](#id-token)
+      + [ID Token sub claim](#id-token-sub-claim)
+   * [Client Authentication](#client-authentication)
+   * [OpenId Foundation Certification](#openid-foundation-certification)
+   * [References](#references)
+
+<!-- TOC end -->
+
+
+## Introduction
+
+This document is the CAMARA Security and Interoperability Profile. To ensure interoperability and increased security, this technical specification restricts some options available in OIDC and CIBA, but does not change these standards.
+
+The CAMARA document sharpens the following for interoperability and security: 
+
+* Of the flows defined in OIDC CAMARA uses the [OIDC Authorization Code Flow](https://openid.net/specs/openid-connect-core-1_0.html#CodeFlowAuth)
+
```

Comment:
* Of the flows defined in Oauth2, Camara uses the [Client credentials grant flow](https://www.rfc-editor.org/rfc/rfc6749#section-4.4)